sources/osgShadow/shaders/tapPCF.glsl

```diff
@@ -1,5 +1,7 @@
 
 
+#pragma include "noise.glsl" "_ROTATEOFFSET"
```

Comment:
Could you improve the coding style with the glsl part and the meaning of _ in defines plz